script/makeinstaller.sh

```diff
@@ -89,6 +91,25 @@ cat >"${build_dir}/installer_tmpl.nsi" <<EOF
 [% endblock %]
 
 [% block install_files %]
+    ; Test if VCRedist 2015 is installed
+    \${If} \${RunningX64}
+        ReadRegStr \$1 HKLM "SOFTWARE\Wow6432Node\Microsoft\VisualStudio\14.0\VC\Runtimes\x86" "Installed"
+        ; if installed, jump past the installer
+        StrCmp \$1 1 installed
+    \${Else}
+        ReadRegStr \$1 HKLM "SOFTWARE\Microsoft\VisualStudio\14.0\VC\Runtimes\x86" "Installed"
+        StrCmp \$1 1 installed
+    \${EndIf}
+
+    ; If not installed run the installer
```

Comment:
Should we also run the x64 installer on 64-bit systems? Using the `vcredist2015` chocolatey package as reference, it looks like we may want to install both x86 and x64 on 64-bit systems